src/BikramSambat.ts

```diff
@@ -87,4 +90,18 @@ export default class BikramSambat {
   public getNextYear(): number {
     return this.year ? this.year + 1 : NaN
   }
+
+  public getWeekdayNames(language: string): string[] {
+    const languageKey = language === 'np' ? 'np' : 'en'
+    return NepaliDaysData.map((day) => day[languageKey])
+  }
```

Comment:
```suggestion
  public getWeekdayNames(language?: string): string[] {
    return NepaliDaysData.map((day) => day[language ?? 'np'])
  }
```

Reply:
sure

Reply:
Im getting type error here while modifying

Reply:
![image](https://github.com/askbuddie/nepali-date-picker/assets/71167057/36b8c398-3c22-4de0-aafe-bb40e23f8795)

Reply:
Element implicitly has an 'any' type because expression of type 'string' can't be used to index type 'Day'.
  No index signature with a parameter of type 'string' was found on type 'Day'.ts(7053)
(parameter) language: string | undefined